lib/primitives/base-element.js

```diff
@@ -17,7 +17,7 @@ module.exports = function(registry) {
         this.attributes.set(attributes);
       }
 
-      this.content = content || null;
+      this.content = content !== undefined ? content : null;
```

Comment:
This covers the issue where content could be falsy, and therefore gets `null`, though you may actually want it to be an empty string..